### DISCUSSION TITLE
How to set default app

### DISCUSSION THREAD

**Discussion by rishujam** [openfoodfacts/openfoodfacts-androidapp#4877] [Q&A]
In the install steps of the readme file, it is written "In Android Studio, make sure to select OFF as the default flavor". How do I do that? In my home fragment, it shows open beauty facts

**Comment by mertcaliskan01** [comment]
Open your project in Android Studio.

Locate the Build Variants panel:
-If you don’t see it, navigate to View > Tool Windows > Build Variants from the top menu.

-In the Build Variants panel, find the app module.

-Next to the app module, there is a dropdown menu showing the current build variant. Click it to see all available flavors combined with build types (e.g., offDebug, obfDebug, etc.).

-Select offDebug (or any off flavor variant) from this list.

-After selecting the OFF flavor, clean and rebuild the project by going to Build > Clean Project and then Build > Rebuild Project.

-Run the app. You should now see the Open Food Facts interface instead of Open Beauty Facts.
